robotbase/src/main/java/frc/robot/commands/intake/IntakeToShooterCommand.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Robot;
+
+public class IntakeToShooterCommand extends Command {
+
+  public IntakeToShooterCommand() {
+    addRequirements(Robot.intake);
+  }
+
+  @Override
+  public void initialize() {}
+
+  @Override
+  public void execute() {
+    Robot.intake.setAxisSpeed(
```

Comment:
setAxisSpeed is for setting the motors from the controller, occasionally extra things are done to the percentOutput that we do not want to do during a normal set within a command. We should have a separate method `set(double topPO, double bottomPO)` in the `Intake` susbsystem for use. It will currently do the same thing the `setAxisSpeed` does. It would be good practice to call the `set` method within the `setAxisSpeed` method as well.

Reply:
You can set the speed once in the `Initialize` method. Setting it in execute will set it every command loop, and will send unnecessary frames on our CAN network.